src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -718,7 +688,7 @@ private class RegexCommentFormatDefinition : ClassificationFormatDefinition
             public RegexCommentFormatDefinition()
             {
                 this.DisplayName = EditorFeaturesWpfResources.Regex_Comment;
-                this.ForegroundColor = s_regexCommentColor;
+                this.ForegroundColor = Color.FromRgb(0x00, 0x80, 0x00);
```

Comment:
should we just leave these statically defined (just with only one option)?

Reply:
I plan to leave it as is for the moment. We don't for any other now and extracting them out later wouldn't be a difficult refactoring if we decided to organize this file in that way.